src/components/shared/TenantSelector.tsx

```diff
@@ -17,7 +17,8 @@ interface Props {
 function TenantSelector({ updateStoreState }: Props) {
     const intl = useIntl();
 
-    const { tenants } = useTenantDetails();
+    const tenantNames = useEntitiesStore_tenantsWithAdmin();
```

Comment:
This is a new hook that is used in just this spot. Since this for sure should not show nested properties we needed a new list.